TACC/protx-dashboard
# Hotifx/redirect and demographics chart height

## Overview: ##

Hotfix to address some open issues:
* Update Readme
* Update redirect for onboarding
* Set max height for demographics chart (It was previously taking up too much vertical space)
* Fixed an issue in demographics chart instructions 
* Make Community Characteristics modal a little shorter

## Related Jira tickets: ##

None

## Summary of Changes: ##

## Testing Steps: ##
1. Check all pages
2. Confirm that community characteristics isn't too large.  It was too tall and we couldn't see the bottom when used in iframe
3. Confirm that when you select a county and view demographics that everything fits on the right panel (ie. demographics chart doesn't take up too much vertical space)
